src/edit/RotateHandle.js

```diff
@@ -10,15 +10,15 @@ L.RotateHandle = L.EditHandle.extend({
 	
 	_onHandleDrag: function() {
 		var overlay = this._handled,
-			formerLatLng = this._handled._corners[this._corner],
+			formerLatLng = overlay._corners[this._corner],
 			newLatLng = this.getLatLng(),
-			angle = this._calculateAngle(formerLatLng, newLatLng);
+			angle = this.calculateAngleDelta(formerLatLng, newLatLng);
 
-	 	overlay.editing._rotateBy(angle);
+	 	if (angle !== 0) { overlay.editing._rotateBy(angle); }
```

Comment:
@sashadev-sky Were there any upsides to adding an if here? Is this to prevent additional computations (i.e., infinite loops on initial state)?

Reply:
Yeah I don't remember specifically if it was causing issues but anyways its good to skip extra calculations in `_rotateBy` esp as we try to figure out how to make rotation and scaling perfect (no distortion)